### DISCUSSION TITLE
Font metrics

### DISCUSSION THREAD

**Discussion by Gurezkiy** [dlemstra/magick-wasm#142] [General]
How to get font metrics? Ascent, descent and others?
https://rj-graffix.com/www/conjure.html

**Comment by dlemstra** [comment]
That api is not yet available but I will make sure to add this soon.

**Comment by hornv** [comment]
Yes, this feature would be very useful.

**Reply by dlemstra** [reply]
I added this at the end of last month and this will become available in the next release.
